spec/assets/manifests-repo/deployments/foo.yml

```diff
@@ -3,3 +3,4 @@ name: foo
 templates:
   - foo/bar.yml
 meta: {}
+stub_value: <%= p('stub.value') %>
```

Comment:
I know erb support has already been added. But wouldn't it make more sense to just inject the stub file into the merge chain and use spiff/spruce for merging in the stub values?